packages/terra-status/src/Status.jsx

```diff
@@ -13,20 +13,20 @@ const propTypes = {
    * Child node. Component to display next to the status indicator.
    */
   children: PropTypes.node.isRequired,
-  /**
-   * The color of the status indicator. Accepts a CSS color value.
-   */
-  color: PropTypes.string,
   /**
    * Visually hidden text used to convey the meaning of the status indicator to screen readers.
    */
   visuallyHiddenText: PropTypes.string,
+  /**
+   * Sets an author defined class, to control the colors of the status indicator
```

Comment:
```suggestion
   * Sets an author defined class, to control the colors of the status indicator.
   * 
   * ![IMPORTANT](https://badgen.net/badge//IMPORTANT/blue?icon=github)
   * Adding `--my-app...` CSS variables is required for proper re-themeability when creating your custom color styles _(see included examples)_.
```
\
\
Adding reminder to use CSS variables:

![IMPORTANT](https://badgen.net/badge//IMPORTANT/blue?icon=github) Adding `--my-app...` CSS variables is required for proper re-themeability when creating your custom color styles _(see included examples)_.